monri/src/main/java/com/monri/android/view/CardNumberEditText.java

```diff
@@ -29,12 +29,13 @@
  */
 public class CardNumberEditText extends MonriEditText {
 
+    private static final int MAX_LENGTH_VISA = 23;
     private static final int MAX_LENGTH_COMMON = 19;
     // Note that AmEx and Diners Club have the same length
     // because Diners Club has one more space, but one less digit.
     private static final int MAX_LENGTH_AMEX_DINERS = 17;
 
-    private static final Integer[] SPACES_ARRAY_COMMON = {4, 9, 14};
+    private static final Integer[] SPACES_ARRAY_COMMON = {4, 9, 14, 19};
```

Comment:
@adnanomerovic explain?

Reply:
spaces are on index 4, 9, 14 and 19 (if visa have 19 character without space, with space max length is 23)

Reply:
Got it. Thanks.